public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -17,7 +17,7 @@ import {
 } from '@grafana/runtime';
 import appEvents from 'app/core/app_events';
 import * as ruleActionButtons from 'app/features/alerting/unified/components/rules/RuleActionsButtons';
-import { mockSearchApi, mockUserApi, setupMswServer } from 'app/features/alerting/unified/mockApi';
+import { mockUserApi, setupMswServer } from 'app/features/alerting/unified/mockApi';
```

Comment:
Whilst cleaning up the `SimplifiedRuleEditor` tests, I needed to add a handler for the `search` endpoint (as that's what drives the folder picking behaviour). 
After doing that, I could remove the previous `mockSearchApi` behaviour in a few places that were only mocking that endpoint out to stop an MSW warning (i.e., they weren't caring about the response of that endpoint, just making sure there wasn't an unhandled endpoint).

I didn't remove all of them as there were a couple of other tests that cared about the specific response, and I didn't want to bloat the PR